dev/com.ibm.websphere.appserver.features/visibility/public/jakartaee-9.0/com.ibm.websphere.appserver.jakartaee-9.0.feature

```diff
@@ -7,8 +7,17 @@ IBM-ShortName: jakartaee-9.0
 Subsystem-Version: 9.0.0
 Subsystem-Name: Jakarta EE Platform 9.0
 -features=\
-  com.ibm.websphere.appserver.concurrent-2.0,\
-  com.ibm.websphere.appserver.webProfile-9.0,\
-  com.ibm.websphere.appserver.transaction-2.0
-kind=noship
-edition=full
+ com.ibm.websphere.appserver.el-4.0, \
+ com.ibm.websphere.appserver.jsp-3.0, \
+ com.ibm.websphere.appserver.servlet-5.0, \
+ com.ibm.websphere.appserver.jsonp-2.0, \
+ com.ibm.websphere.appserver.jsonb-2.0, \
+ com.ibm.websphere.appserver.transaction-2.0, \
+ io.openliberty.jakarta.cdi-3.0, \
+ io.openliberty.jakarta.interceptor-2.0, \
+ com.ibm.websphere.appserver.jdbc-4.0, \
+ com.ibm.websphere.appserver.jdbc-4.1, \
+ com.ibm.websphere.appserver.jdbc-4.2, \
+ com.ibm.websphere.appserver.jdbc-4.3
```

Comment:
I don't think you can pull in multiple versions of jdbc like this... you need to pull in just 4.3 and then optionally tolerate the others. ee8 pulls in 4.2 and tolerates 4.3 ... this should be the opposite I think?

Reply:
This was fixed...